react/week2/homework/src/ToDoList.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import Todo from "./ToDo";
+
+class TodoList extends Component {
+	constructor() {
+		super();
+		this.state = {
+			todoItems: [
+				{ id: 1, description: "Get out of bed" },
+				{ id: 2, description: "Brush teeth" },
+				{ id: 3, description: "Eat breakfast" },
+			],
+		};
+		this.addTodo = this.addTodo.bind(this);
+		this.deleteTodo = this.deleteTodo.bind(this);
+	}
+    addTodo = () => {
+        const newItem = { id: this.state.todoItems.length + 1,description: "random text" };
```

Comment:
What will happen if you delete todo with id 2, "Brush teeth" for example? And then add a new todo, which ID will have the new todo? 
Hint: Warning: Encountered two children with the same key. Keys should be unique so that components maintain their identity across updates.

Reply:
Thank you for asking. I didnt think about that scenario. I had tried with Math.max command
s but didnt work so i used uuid

Reply:
I have implemented the changes as requested. Kindly review